apple/MarkdownTextInputDecoratorComponentView.mm

```diff
@@ -116,17 +83,19 @@ - (void)didMoveToWindow {
 
 - (void)willMoveToWindow:(UIWindow *)newWindow
 {
-  if (_textInput != nil) {
-    [_textInput setMarkdownUtils:nil];
-  }
-  if (_adapter != nil) {
-    [_adapter setMarkdownUtils:nil];
-  }
-  if (_textView != nil) {
-    [_textView setMarkdownUtils:nil];
-    if (_textView.layoutManager != nil && [object_getClass(_textView.layoutManager) isEqual:[MarkdownLayoutManager class]]) {
-      [_textView.layoutManager setValue:nil forKey:@"markdownUtils"];
-      object_setClass(_textView.layoutManager, [NSLayoutManager class]);
+  if (newWindow == nil) {
```

Comment:
Why do we need to add this conditional? Can we use `if (newWindow != nil) { return; }` instead to keep current levels of indentation for the sake of smaller diff of this PR?

Reply:
We can - [05a7fe4](https://github.com/Expensify/react-native-live-markdown/pull/566/commits/05a7fe40f506a301e6fd6dc48e0eb2511d5d0d11).

As to why we need this - `willMoveToWindow` is called when the view is mounted and unmounted, looks like mount happens **after** props are set for the first time before mounting the host view - it was effectively cleaning up during mount.